build/shards.zsh

```diff
@@ -0,0 +1,25 @@
+
+function shards.build {
+	write "${target}: ${src[@]} ${depends[$target]}"
+	write "\t@echo '${fg_bold[magenta]}  [SHARD]    ${fg_bold[white]}$target${reset_color}'"
+	write "\t${Q}shards build"
+
+	write "\n"
+}
+
+function shards.install {
+	binary.install "$@"
+}
+
+function shards.uninstall {
+	binary.uninstall "$@"
+}
+
+function shards.clean {
+	script.clean "$@"
+}
+
+function shards.distfiles {
+	echo "${target}"
```

Comment:
shards will rely on things from lib/ and its shard.yml. lib/ can be ignored if you run `shards update` somewhere, but shard.yml will be needed, and only once.